.github/workflows/deploy-cloud.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
           config-files: values.production.yaml
           chart-path: charts/budibase
           namespace: budibase
-          values: globals.appVersion=v${{ env.RELEASE_VERSION }}
+          values: globals.appVersion=v${{ env.RELEASE_VERSION }},services.couchdb.backup.target=${{ secrets.PRODUCTION_COUCHDB_URL }},services.couchdb.url=${{ secrets.PRODUCTION_COUCHDB_URL }},services.couchdb.password=${{ secrets.PRODUCTION_COUCHDB_PASSWORD }}
```

Comment:
No need for `services.couchdb.user`?

Reply:
That one gets updated in `budibase-infra` - thought it pointless to add it here as it's not really a secret compared to the password